<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YoshiSO</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Descargas/Yoshi.png</normaloff>../../Descargas/Yoshi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="IniProcGP">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Procesos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="noProcLE">
             <property name="placeholderText">
              <string>No. Procesos</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="noProcePB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageResPro">
       <widget class="QLabel" name="unfinishedBatchLB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>8</y>
          <width>311</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Lotes Pendientes: </string>
        </property>
       </widget>
       <widget class="QGroupBox" name="executingprocGP">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>70</y>
          <width>341</width>
          <height>671</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="idLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="onIdLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="operationLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Operaci√≥n:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="onOperatorLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="estTimeLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tiempo estimado:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="onEstimatedTimeLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="passTimeLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tiempo transcurrido: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="onPassTimeLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="restTimeLB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tiempo restante: </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="onRestTimeB">
           <property name="font">
            <font>
             <family>Rockwell</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>67</y>
          <width>311</width>
          <height>681</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="pendingProcGL"/>
         </item>
        </layout>
       </widget>
       <widget class="QScrollArea" name="finishedProcSA">
        <property name="geometry">
         <rect>
          <x>786</x>
          <y>63</y>
          <width>311</width>
          <height>681</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>307</width>
           <height>677</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="finishedProcGL"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="startPB">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>0</y>
          <width>331</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QLabel" name="globalCountLB">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>8</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contador global:</string>
        </property>
       </widget>
       <widget class="QLabel" name="pendingProcLB">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>42</y>
          <width>311</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Procesos pendientes</string>
        </property>
       </widget>
       <widget class="QLabel" name="executingProcLB">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>40</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Procesos ejecutandose</string>
        </property>
       </widget>
       <widget class="QLabel" name="finishedProcLB">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>40</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Procesos finalizados</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
